src/components/PhotoPage/PhotoPage.js

```diff
@@ -24,79 +24,71 @@ const teamIntro = `
 const officers = [
 	{
 		role: 'President',
-		name: 'Lea Blum',
-		id: 'lea',
-		description: `
-			Hi there, I’m Lea and I’m one of the Hack Co-Presidents (along with
-			Connie). Come hang out with me and the rest of hack at our events to hear
-			my (sometimes) funny jokes and (always) hilarious puns.
-		`
+		name: 'Raji Jadhav',
+		id: 'raji',
+		description: `Hey! I'm Raji, a 2nd year from 
+		India! Love to dance, love to sing, love to code. I believe in 
+		community learning, a multiverse, and fairies :) Current obsessions 
+		include ML, Thai food, and calligraphy. Tend to never stop talking.`
 	},
 	{
-		role: 'President',
-		name: 'Connie Chen',
-		id: 'connie',
-		description: `Hello! I'm Connie, a third year computer science major. 
-		I love the california weather, facebook messenger stickers, and music. 
-		Getting added to the Dogspotting fb group also changed my life (shout-out to my co Lea).`
+		role: 'Vice-President',
```

Comment:
In the US they spell it without the hyphen.

```suggestion
		role: 'Vice President',
```